l10n_co_geo/models/res_country_state.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#    License, author and contributors information in:                         #
+#    __openerp__.py file at the root folder of this module.                   #
+###############################################################################
+
+from openerp import models, fields
+
+
+class ResState(models.Model):
+    _name = 'res.country.state'
+    _inherit = 'res.country.state'
+
+    iso = fields.Char(
```

Comment:
The same: I don't know for what do you use this, but normally it's possible to split things, or just don't depend on the module and detect dynamically if the fields are present.